test/dynamo/test_misc.py

```diff
@@ -7074,6 +7075,23 @@ def fn(it):
         self.assertEqual(list(eager), list(compiled))
         self.assertEqual(counter.frame_count, 1)
 
+    @unittest.skipIf(not TEST_NUMPY, "NumPy not found")
+    @torch._dynamo.config(trace_numpy=True)
+    def test_trace_ndarray_frame(self):
+        import numpy as np
```

Comment:
NumPy is already imported globally. It's fine to import NumPy globally in tests, but not in the `torch/` folder.